spec/features/users/login_spec.rb

```diff
@@ -93,4 +93,45 @@
       expect(page).to have_content("Welcome, #{merchant.name}, you are logged in!")
     end
   end
+  describe "all users can log out" do
+      it 'can log out as a registered user' do
+      user = User.create(name: "Jordan",
+                          address: "394 High St",
+                          city: "Denver",
+                          state: "CO",
+                          zip_code: "80602",
+                          email: "hotones@hotmail.com",
+                          password: "password",
+                          password_confirmation: "password",
+                          role: 0)
+
+      visit '/'
```

Comment:
We can just do the stubs thing here: 
allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
instead of having to do the whole login rigamorale.

Reply:
Good use of 'rigmarole' @madelynrr